apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -211,6 +211,9 @@ export DATADOG_REDIS_CONFIG ?= foo
 # see https://kubernetes.io/docs/concepts/configuration/secret/
 export DATADOG_REDIS_CONFIG_BASE64 ?= $(shell echo -n "${DATADOG_REDIS_CONFIG}" | base64)
 
+export API_RATELIMIT_ENABLE ?= false
+export WEB_RATELIMIT_ENABLE ?= true
```

Comment:
A tiny nit, but would you mind moving each of these up into their respective groups?